lxd/device/nic.go

```diff
@@ -43,6 +43,7 @@ func nicValidationRules(requiredFields []string, optionalFields []string, instCo
 		"ipv6.host_address":                    validate.Optional(validate.IsNetworkAddressV6),
 		"ipv4.host_table":                      validate.Optional(validate.IsUint32),
 		"ipv6.host_table":                      validate.Optional(validate.IsUint32),
+		"txqlen":                               validate.Optional(validate.IsUint32),
```

Comment:
Can we call this `queue.tx.length` please for clarity and consistency with existing settings naming scheme.

Reply:
As I understand we want txqlen be changeable by calling `lxc config device set c1 eth0 txqueuelen=100`
and if I change "txqlen" to "queue.tx.length" it does not work like that.  So should I change it to `queue.tx.length` or leave `txqueuelen`?

Reply:
I think using `lxc config device set c1 eth0 queue.tx.length=100` would be more inline with the other settings, and would allow us to potentially accommodate other queue related settings in the future.

Reply:
Any preference here @stgraber ?

Reply:
Lets go with `lxc config device set c1 eth0 queue.tx.length=100` then @Viktor-Yakovchuk

Reply:
Done